src/Pages/Footer/Footer.css

```diff
@@ -178,6 +187,24 @@
       margin-left: 0px !important;
     }
 
+    .grid_container__box1__text_container__email>div{
+        visibility: hidden;
+    }
+
+    .grid_container__box4{
+        display: block
+    }
+    .grid_container__box4 > div{
+        display: flex;
+        justify-content: center;
+     }
+     .grid_container__box4 > div > svg{

```

Comment:
here if we change margin-right:40px;
 
to margin:0xp 20px; that changes the SVG icon in center now svg icons are in right size

Reply:
@omernadeem1070

Reply:
I'll check